src/main/java/strategy/Strategy.java

```diff
@@ -0,0 +1,15 @@
+package strategy;
+
+import dto.ShopOperation;
+import java.util.List;
+import java.util.Map;
+
+public class Strategy {
+    public void performOperation(List<ShopOperation> shopOperationList,
+                                 Map<String, OperationHandler> handlers) {
```

Comment:
let's pass `handlers` not in method parameters, but make it a field and initialize in the constructor